src/diffusers/__init__.py

```diff
@@ -17,7 +17,7 @@
 
 if is_torch_available():
     from .modeling_utils import ModelMixin
-    from .models import AutoencoderKL, UNet2DConditionModel, UNet2DModel, VQModel
+    from .models import AutoencoderKL, Transformer, UNet2DConditionModel, UNet2DModel, VQModel
```

Comment:
```suggestion
    from .models import AutoencoderKL, UNet2DConditionModel, UNet2DModel, VQModel
```

Let's not make this public - very high chance of being changed down the road

Reply:
Removing that from the init gives the following error:
```
>>> pipe = AbsorbingDiffusionPipeline.from_pretrained("nielsr/absorbing-diffusion-churches")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/nielsrogge/Documents/python_projecten/diffusers/src/diffusers/pipeline_utils.py", line 363, in from_pretrained
    class_obj = getattr(library, class_name)
AttributeError: module 'diffusers' has no attribute 'Transformer'
```

Reply:
Ah ok, sorry it's a whole new model class similar to `UNet2D....`! Ok then we should leave it public and I have to dive into the API there a bit more since it's fully public once we merge this.

Do you think `Transformer` is a good name? Should we maybe call it `Transformes2D` (since we'll surely have transformer-like audio models that will differ heavily from this one in the future)